src/transformers/pipelines/token_classification.py

```diff
@@ -85,6 +85,7 @@ def __init__(
         ignore_labels=["O"],
         task: str = "",
         grouped_entities: bool = False,
+        subword_label_re_alignment: Union[bool, str] = False,
```

Comment:
I wonder if `aggregate_subwords` would be a more suitable name?

Reply:
I would understand `aggregate_subwords` better than `subword_label_re_alignment`

Reply:
Or would `aggregate_strategy` be even better, as we're actually prompting for a strategy?

Reply:
Having this accept enum parameters as value would be great, similar to what we do with `PaddingStrategy`:

https://github.com/huggingface/transformers/blob/9f4e0c23d68366985f9f584388874477ad6472d8/src/transformers/file_utils.py#L1657-L1665